docs/HISTORY.txt

```diff
@@ -4,7 +4,8 @@ Changelog
 5.0.4 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- use `plone.api.portal.get` instead of `getToolByName`
+  [vangheem]
```

Comment:
minor quibble: you're actually replacing `getPortal`

Reply:
`getPortal` is the internal method; however, it used an implementation of `getToolByName` to get the portal object.